Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -145,14 +146,18 @@ public LibraryManager(
             _mediaEncoder = mediaEncoder;
             _itemRepository = itemRepository;
             _imageProcessor = imageProcessor;
-            _cache = new ConcurrentDictionary<Guid, BaseItem>();
+            _cache = new ConcurrentLruBuilder<Guid, BaseItem>()
```

Comment:
Please provide that via DI instead

Reply:
I disagree. Not everything needs to be resolved from DI. This is a local cache

Reply:
i see no reasons why this could not be shared between other components and/or maybe even used for a 2nd level database cache

Reply:
What is the benefit of sharing it? If another component needs a cache they can just new up their own?

Reply:
I tried adding the second level cache interceptor back in and performance was much worse. I don't see any place that we could use a shared cache right now

Reply:
Fair enough.
Thought it might just give us _the option_ to reuse it later if its already there.